cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -360,3 +360,83 @@ def test_undefined_register_from_register_arg():
 
     with pytest.raises(QasmException, match=r"""Undefined.*register.*q.*"""):
         parser.parse(qasm)
+
+
+def test_measure_individual_bits():
+    qasm = """
```

Comment:
Add a failure case where the output register is quantum, and another where the measured register is classical.